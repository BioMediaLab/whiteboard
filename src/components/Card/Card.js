import { Card } from '@shopify/polaris'

export default Card
