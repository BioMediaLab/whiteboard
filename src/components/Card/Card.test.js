import React from 'react'
import Card from './'

describe('Card', () => {
  test('can render', () => {
    const component = render(
      <Card>
        <p>Here goes the card!</p>
      </Card>
    )

    expect(component).toMatchSnapshot()
  })
})
