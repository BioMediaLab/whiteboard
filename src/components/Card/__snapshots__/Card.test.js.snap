// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card can render 1`] = `
<div>
  <div
    class="Card"
  >
    <p>
      Here goes the card!
    </p>
  </div>
</div>
`;
